usaspending_api/api_docs/api_documentation/awards/awards.md

```diff
@@ -1,13 +1,13 @@
 ## Awards
-**Route:** `/api/v2/awards/<generated_unique_award_id>/`
+**Route:** `/api/v2/awards/<requested_award>/`
```

Comment:
It took a few names before I decided on `requested_award`. 

The reason I didn't include `id` in the name is because there are different types of "IDs" accepted there. Originally it was only `generated_unique_award_id`, now it will accept either `generated_unique_award_id` or `id` and will function without any hints or additional parameters.

My thinking was if the field name is generic enough, API users won't accidentally believe there is a new ID in the DB or an API response called `requested_award_id`